jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ContextHandler.java

```diff
@@ -2607,6 +2607,7 @@ protected void doStop() throws Exception
         @Override
         public void insertHandler(Singleton handler)
         {
+            // reimplemented insertHandler so we can directly call super.setHandler
```

Comment:
Supposed to be a TODO? Otherwise it's unclear what the comment means.

Reply:
`o.e.j.ee9.nested.ContextHandler.setHandler` is not meant to be called directly and gives a warning that it should not be used.   So this code is exactly the super.insertHander code, but with `super.setHandler` called instead.

Reply:
replaced with:
```java
            // We cannot call super.insertHandler here, because it uses this.setHandler
            // which gives a warning.  This is the same code, but uses super.setHandler
```